### ISSUE TITLE
Messages / Stall Messages

### ISSUE THREAD

**Issue by bitcoinbekka** [PlebeianApp/plebeian.market#426]
Is there a separate messaging section for the market stall/products? 

It can get confusing and messy with all the Nostr messaging displayed, when they are not related to the market stall or products that are listed

**Comment by gzuuus** [comment]
Yes, there will be a separation between marketplaces and stalls, but currently, this is not possible due to the limitations of the current NIP15 spec. Unfortunately, the spec relies heavily on direct messages (kind04) for all interactions, which is not ideal. We are already working on updating the NIP15 spec to address these issues and resolve the existing problems. Until the updated spec is released, we are limited in what we can do.
